README.md

```diff
@@ -192,6 +192,25 @@ This could be used like so: `crane provision service1`, `crane run -v databases`
 
 When using targets, it is also possible to cascade the commands to related containers. There are 2 different flags, `--cascade-affected` and `--cascade-dependencies`. In our example configuration above, when targeting the `mysql` container, the `apache` container would be considered to be "affected". When targeting the `apache` container, the `mysql` container would be considered as a "dependency". Both flags take a string argument, which specifies which type of cascading is desired, options are `volumesFrom`, `link`, `net` and `all`.
 
+### YAML advanced usage
+YAML gives you some advanced features like [alias](http://www.yaml.org/spec/1.2/spec.html#id2786196) and [merge](http://yaml.org/type/merge.html). They lets you a way to easily avoid duplicated code in your `crane.yml` file. As a example, imagine you need to define 2 different webapps: `web` and `admin`. They share almost the same configuration but the `cmd` declaration. And imagine you also need 2 instances for each one for using with a node balancer. Then you can declare them as simply:
```

Comment:
"And imagine you also need 2 instances for each one for using with a node balancer. Then you can declare them as simply:" -> "In case you need several instances of the same image to use them behind a load balancer for example, you can simply declare them as:"